spec/features/post/show.html.erb_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :feature do
+  describe 'test' do
+    before :each do
```

Comment:
- [ ] Here you need to also add a test  that asserts you can see the username of each commenter.

Reply:
- [ ] @team kindly add test to show you can see the username of each commenter as previously requested